cmake/projects/NASM/schemes/url_sha1_nasm_windows.cmake.in

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2017, Zhuhao Wang
```

Comment:
> I want to install the unpacked binaries into bin

I think it's responsibility of package itself. When we set `configure --prefix=some-dir` then `make install` should install it to `some-dir/bin`. If this scheme doesn't fit this package, then keep using your custom.